source/elements/oneDAL/source/bibliography.rst

```diff
@@ -5,3 +5,14 @@ Bibliography
 ============
 
 For more information about algorithms implemented in |dal_short_name|, refer to the following publications:
```

Comment:
[dal_short_name] again

Reply:
this is just an rst replacement for oneDAL, in the output it replaces with the appropriate name 
![image](https://user-images.githubusercontent.com/17865025/77143638-d6393980-6a94-11ea-8ade-847e0fc32c0c.png)

Reply:
Okay
My concern is on the correct use of the short names, e.g, we start intro section with the short name while I'd start with full name and then switch to the short, per legal guidelines. Did the guidelines change?

Reply:
or, in the specs we should not use Intel specifics?

Reply:
the first usage should be spelled out
but in the document, I believe? not in each topic

Not sure about spec's specific guidelines

Reply:
I'll double-check this q with Robert today

Reply:
Yes, not in each topic